.github/workflows/build.yml

```diff
@@ -100,6 +100,10 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: 3.x
+    - name: Install Aspell
```

Comment:
Linting doesn't need to install Aspell, that is why I prefer to have it as a separate action. I think it is more clear when it is separate.